priv/perf/apps/load_test/lib/scenario/deposits.ex

```diff
@@ -0,0 +1,129 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Scenario.Deposits do
+  use Chaperon.Scenario
+
+  alias Chaperon.Session
+  alias LoadTest.ChildChain.Deposit
+  alias LoadTest.Ethereum
+  alias LoadTest.Ethereum.Account
+  alias LoadTest.Service.Faucet
+
+  @spec run(Session.t()) :: Session.t()
+  def run(session) do
+    tps = config(session, [:run_config, :tps])
+    period_in_seconds = config(session, [:run_config, :period_in_seconds])
+
+    total_number_of_transactions = tps * period_in_seconds
+    period_in_mseconds = period_in_seconds * 1_000
+
+    session
+    |> cc_spread(
+      :create_deposit,
+      total_number_of_transactions,
+      period_in_mseconds
+    )
+    |> await_all(:create_deposit)
+  end
+
+  def create_deposit(session) do
+    token = config(session, [:chain_config, :token])
+    deposited_amount = config(session, [:chain_config, :deposited_amount])
+    initial_amount = config(session, [:chain_config, :initial_amount])
+    transferred_amount = config(session, [:chain_config, :transferred_amount])
+
+    {:ok, from_address} = Account.new()
+    {:ok, to_address} = Account.new()
+    {:ok, _} = Faucet.fund_root_chain_account(from_address.addr, initial_amount)
+
+    txhash =
+      Deposit.deposit_from(from_address, deposited_amount, token,
+        return: :txhash,
+        deposit_finality_margin: 10,
+        gas_price: 0
+      )
+
+    gas_used = Ethereum.get_gas_used(txhash)
+
+    with :ok <-
```

Comment:
1. check childchain balance after deposit for the depositor
2. check rootchain balance after the deposit for the depositor
3. send value to another account
4. check childchain value for the receiver

Reply:
leave this as a comment! it's good!

Reply:
I added it to the moduledoc